app/src/components/Home.vue

```diff
@@ -0,0 +1,76 @@
+<style lang="scss">
+
+/**
+ *--------------------------------------------------------------------------
+ *	Vue-component for home
+ *--------------------------------------------------------------------------
+ *
+ * 	This is the Vue-component for home.
+ *
+ * 	This file includes the HTML template, sass-stylesheet 
+ * 	and javascript for home.
+ *
+ */
+
+.home{
+	.row{
+		.home-header-titles{
+			margin-bottom:60px;
+
+			h1{
+				margin:0;
+			}
+
+			h2{
+				margin-top:0;
+				margin-bottom:30px;
+			}
+		}
+	}
+}
+
+</style>
+
+<template>
+	<div class="container home">
+		<div class="row">
+			<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
```

Comment:
@OezgenVolkan Wenn du col-xs-12 gesetzt hast, gilt col-sm-12, col-md-12, col-lg-12 automatisch auch, solange du das nicht änderst.
Deshalb würde ich das in Zukunft lieber weg lassen.